lib/omniship/usps/track/package.rb

```diff
@@ -37,9 +37,11 @@ def has_left?
         def has_arrived?
           # http://about.usps.com/publications/pub97/pub97_i.htm
           activity.any? {|activity|
-            # deliverred or ready for pickup at post office, or Notice Left 
-            ["01", # Delivered*
+            # deliverred or ready for pickup at post office, or Notice Left
+            [
+              "01", # Delivered*
               "17", # Picked Up By Agent, Tendered to Returns Agent
+              "07", # Arrived at Unit
```

Comment:
my research and the examples I was able to pull do not lead me to believe that this is supposed to mean the customer has it

Reply:
^^^